database/migrations/Version20240416134850.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Database\Migrations;
+
+use Doctrine\DBAL\Schema\Schema;
+use Doctrine\Migrations\AbstractMigration;
+
+final class Version20240416134850 extends AbstractMigration
+{
+    public function getDescription(): string
+    {
+        return 'Renames generic_field1 to custom_field1.';
```

Comment:
Technically, given that we haven't released anything yet, we don't need this middle migration to change the name - we could just edit database/migrations/Version20240328114317 - but that said, no big deal - let's just keep it as it is, if it's working.